tests/performance/io_uring.xml

```diff
@@ -0,0 +1,13 @@
+<test>
+    <settings>
+        <local_filesystem_read_method>io_uring</local_filesystem_read_method>
+    </settings>
+
+    <create_query>CREATE TABLE hits_none (WatchID UInt64 CODEC(NONE)) ENGINE = MergeTree ORDER BY tuple()</create_query>
+    <fill_query>INSERT INTO hits_none SELECT WatchID FROM test.hits</fill_query>
+    <fill_query>OPTIMIZE TABLE hits_none FINAL</fill_query>
```

Comment:
what is the purpose of this? i.e. why not just read from `test.hits`?

Reply:
oh, sorry, I didn't noticed `CODEC(NONE)`. then the question is why we need codec `None`?)

Reply:
probably a bad copy paste, as I dont see a real reason for it, fixed